v2/tools/generator/internal/conversions/typed_conversion_endpoint.go

```diff
@@ -54,3 +62,20 @@ func (endpoint *TypedConversionEndpoint) IsBagItem() bool {
 	_, result := AsPropertyBagMemberType(endpoint.Type())
 	return result
 }
+
+func (endpoint *TypedConversionEndpoint) WithPath(
+	path []astmodel.PropertyName,
+) *TypedConversionEndpoint {
+	var builder strings.Builder
+	for i, name := range path {
```

Comment:
minor: You could probably have used `strings.Join` here, though it would require transforming the path to strings first.

Reply:
Changed.